frontend/src/tests/CreateFolder.spec.js

```diff
@@ -0,0 +1,30 @@
+import Enzyme, { configure } from "enzyme";
+import React from "react";
+import Adapter from "enzyme-adapter-react-16";
+import fetch from "node-fetch";
+import { mount} from "enzyme";
+import CreateFolderContent from "../main-components/CreateFolderContent";
+
+Enzyme.configure({ adapter: new Adapter() });
+global.fetch = fetch;
+
+describe('Creating Folder Page testing', () => {
+  test("When typing folderName live preview is available", () => {
+    const wrapper = mount(<CreateFolderContent />);
+    wrapper.find('#folderName').simulate('blur', { target: { value: 'Verbs' } });
+    expect(wrapper.find('#previewFolderName').text()).toEqual("Verbs")
+  });
+
+  test("When selecting a language live preview is displayed", () => {
+    const wrapper = mount(<CreateFolderContent />);
+    wrapper.find("#language").simulate("change", { target: { value: "Spanish" } });
+    expect(wrapper.find("#previewFolderLang").text()).toEqual("Spanish");
+  });
+  // TODO: When our website is fully completed we should be rerouted to an empty create Folder page
```

Comment:
TODOs should be accompanied by an assignee or a bug. A bug is preferred.

```
// TODO(#123): ...
```

or

```
// TODO(esaracay): ...
```